### ISSUE TITLE
setTags property does not exist on Client type

### ISSUE THREAD

**Issue by georgeperry1** [MindscapeHQ/raygun4node#103]
When using the `setTags` property in a Typescript file, I receive the following type error:

```
Property 'setTags' does not exist on type 'Client'.ts(2339)
```

Here is my code:

```
import raygun from 'raygun'

type UserSettings = {
  identifier: string,
  isAnonymous: boolean
}

const apiKey: string = process.env.RAYGUN_API_KEY || ''
const client: Client = new raygun.Client().init({ apiKey })

const raygunServerWrapper = (tags: string[], userSettings: UserSettings) => {
  client.setVersion(process.env.APP_VERSION || '')
  client.user = () => ({
    ...userSettings
  })

  if (tags) {
    client.setTags(tags)
  }

  return client
}

export default raygunServerWrapper
```

When checking the Client type file, it looks like `setTags` is missing. Not sure if this is by design or a small bug?

<img width="492" alt="Screenshot 2021-03-29 at 11 42 22" src="https://user-images.githubusercontent.com/32910464/112825670-f4f8e580-9083-11eb-9c41-2ef070f37163.png">

I'm happy to make a PR to add it in if required.

**Comment by Widdershin** [comment]
Hi @georgeperry1,

We have a change awaiting release that should resolve this. We should have a new version out in the next few days.

Thanks for the detailed report and offer of a PR 🙂

**Comment by georgeperry1** [comment]
Thanks for the quick update!

**Comment by Widdershin** [comment]
@georgeperry1 

You're welcome 😄 

We've released v0.12.4 today with better TypeScript support.

Types can now be imported like this:

`import * as Raygun from 'raygun'`;

or

`import { Client } from 'raygun'`;

Unfortunately we don't support the default style of imports, as that would require a breaking change.

**Comment by miquelbeltran** [comment]
This should be fixed now, the generated raygun.d.ts provides the `setTags` method.
